cnxarchive/database.py

```diff
@@ -112,6 +113,60 @@ def initdb(settings):
             for schema_part in get_schema():
                 cursor.execute(schema_part)
 
+            # If you are connecting to a database that is not localhost,
```

Comment:
May be the logic should be reversed.  Right now, it's checking whether the database is local and if it is, check whether we're using virtualenv.  

It makes more sense to me if we check whether we're using virtualenv first, and if the database is remote, raise an exception, if the database is local, set up virtualenv for postgres.  If we're not using virtualenv, whether the database is remote or not, there's nothing to do and no errors to raise.